lib/blip.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+p = process;
+e = p.env;
+// for debugging..
+q = // function(i) { console.log(Error(i).stack); p.exit(0); } ||
+  p.exit.bind(p,0);
+r = require;
+t = setTimeout(q, 1000);
+t.unref && t.unref();
+try {
+  r('fs').statSync('.git').isDirectory() && q('skip');
+} catch(x) {
+  r('http').get({
+    // hostname: 'requestb.in',
+    hostname: 'blip.strongloop.com',
+    path: // '/sxy98jsx?' +
```

Comment:
They are place-holders for if someone wants to make changes to the script and run it against http://requestb.in